src/h3lib/lib/h3Index.c

```diff
@@ -557,6 +557,11 @@ H3Index _faceIjkToH3(const FaceIJK* fijk, int res) {
 
     // check for res 0/base cell
     if (res == 0) {
+        if (fijk->coord.i > 2 || fijk->coord.j > 2 || fijk->coord.k > 2) {
+            // out of range input
+            return 0;
```

Comment:
is this a new sentinel value? if it is, can we add it to a documentation? if it isn't new, never mind :)

Reply:
This is the same sentinel used in geoToH3, which is why I used it here.